Views/Home/Resources.cshtml

```diff
@@ -29,8 +34,8 @@
         </a>
     </li>
     <li>
-        <a href="xmpp:codingteam@conference.jabber.ru?join">
-            <i class="fa-li fa fa-lightbulb-o"></i> XMPP conference
+        <a href="https://telegram.me/joinchat/BE1AdEAiwR85TsuRlhU_bA">
+            <i class="fa-li fa fa-envelope-o"></i> Telegram group
```

Comment:
Sorry, I'm late to the show, but why did you choose `fa-envelope-o` over [`fa-telegram`](http://fontawesome.io/icon/telegram/)?